tools/tests/package-tests.js

```diff
@@ -1952,3 +1958,637 @@ selftest.define("show package w/many versions",
     ]
   });
  });
+
+
+// This tests that we get the right excerpt out of the Readme.md in different
+// combinations. It doesn't test publication, because publishing is slow --
+// that's covered in a different test.
```

Comment:
We don't really test internal functions with the current sandbox framework -- we do sometimes in old tests, and it has caused us a lot of headaches: sometimes files are initialized before/after other files, etc. Do you think that this is the time to break that invariant? I'd rather not... (Though possibly hope to have actual unit tests later)